lib/modules/versioning/distro.spec.ts

```diff
@@ -1,7 +1,17 @@
+import { DateTime, Settings } from 'luxon';
 import { DistroInfo } from './distro';
 
 describe('modules/versioning/distro', () => {
+  const dt = DateTime.fromISO('2022-03-20');
+
+  // DistroInfo constructor is dependent on the current date
```

Comment:
we can't discard versions on constructor, as the object can long live in memory (multiple days).

Reply:
ive removed all time mocks for distro.ts,
one caveat is that i had to add one `istanbul ignore` comment.